tasks/e2e-kitchensink.sh

```diff
@@ -146,6 +146,10 @@ PORT=3001 \
   nohup yarn start &>$tmp_server_log &
 grep -q 'You can now view' <(tail -f $tmp_server_log)
 
+# We haven't ejected yet which means there's no `babel` key
+# in package.json yet
+echo '{"presets":["react-app"]}' > .babelrc
```

Comment:
Can we just put it in our jest integration config or something? Why does it have to be at the root?

Reply:
I dunno what's best, but during eject we add it to `package.json`. Will file follow up issue.

Reply:
I just mean that in this case we only need it for integration test itself. So it's unrelated to our app and I'd like it to stay unrelated (if that makes sense)

Reply:
That makes sense. We'll figure something out.